### DISCUSSION TITLE
Bug: When adding multiple reviewers from New PR sreen, shows "removed review request"

### DISCUSSION THREAD

**Discussion by sandipndev** [community/community#8939] [Pull Requests]
### Context

When making a new PR, it sometimes show some people I might want to request review from (probably based on who changed those files in the last few days).

There is a **Request** button beside their name.

### The Bug

If you click on the Request button for *more than 1* expected reviewers one after another quickly, in the PR History UI, it shows that:

> `person1` requested review from `person2` and `person3` and removed request for `person2`

And that's what happens as well however we never removed review request from `person2`

### Expected Behaviour

The PR History should show just:

> `person1` requested review from `person2` and `person3`

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by dennishuo** [comment]
For posterity - I possibly ran into a variation of this issue in the case of "re-request review" from `person2` causing `person3` to be removed entirely, with the UI history showing

`person1` requested review from `person2` and removed request for `person3`

when it doesn't even appear that I have repository permissions for assigning or removing reviewers at all (only a button for "re-requesting" reviews).

I'll separately file a bug report as well.
